### PR TITLE
Add RFC outlining e-resource authentication

### PR DESCRIPTION
**Pull request by kenoir**
[Markdown preview](https://github.com/wellcomecollection/docs/blob/eresource-review/rfcs/XXX-eresource_authentication/README.md)

Very much WIP.

### CODE REVIEW

File: rfcs/XXX-eresource_authentication/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+The URL indexed is for the [EBSCO](https://www.ebsco.com/) Link Resolver
+                           
+EBSCO is a service used by "Collections and Research" to handle database/e-journal metadata.
+
+## Current access methods
+
+### Staff
+
+Access for staff is via [Shibboleth](https://en.wikipedia.org/wiki/Shibboleth_Single_Sign-on_architecture), a profile of [SAML](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language).
```
Comment (kenoir) [inline rfcs/XXX-eresource_authentication/README.md:27]: Gonna have to explain shibboleth somewhere with a request diagram and also discuss federations and publisher requirements for membership - athens/openathens and the relationship to shibboleth.

File: rfcs/XXX-eresource_authentication/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Staff access will be unaffected by the shut down of the Wellcome Library site. 
+
+### Patrons
+
+Library patrons can get access to e-resources by the following methods.
+
+#### Kiosk access
+
+Patrons who are on the library premises can get access from "kiosk" computers that sit on the Wellcome Collection network.
```
Comment (kenoir) [inline rfcs/XXX-eresource_authentication/README.md:39]: Clarify that these computers are actually sitting in the library building.

File: rfcs/XXX-eresource_authentication/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+As part of decommissioning the Wellcome Library site we must ensure continuity of access to those resources for patrons and staff.
+
+## E-resources in the Catalogue API
+
+E-resources are currently available from the page linked above, but [can also be found](https://api.wellcomecollection.org/catalogue/v2/works?query=awa6c6gm&include=items) in the catalogue API.
+
+Items will have an `online-resource` location:
+
+![online resource item](online_resource.png)
```
Comment (alexwlchan) [inline rfcs/XXX-eresource_authentication/README.md:17]: nit: can you make this JSON? Easier to read in Markdown, search for, more accessible than an image.

File: rfcs/XXX-eresource_authentication/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Kiosk access
+
+Patrons who are on the library premises can get access from "kiosk" computers that sit on the Wellcome Collection network.
+
+Access to e-resources is achieved by providing publishers / database providers with a fixed IP range from which resources will be accessed. 
+
+The kiosk access method must be maintained and should be unaffected by the shut down of the Wellcome Library site.
+
+The EBSCO link resolver URL that is referenced in Catalogue API results will provide you with a WAM Proxy link that does not require authentication.
```
Comment (alexwlchan) [inline rfcs/XXX-eresource_authentication/README.md:45]: 1. What is a WAM Proxy? What is a WAM Proxy link?
2. I'm confused – the way this paragraph was ordered, I thought the fixed IP range was for the library building. If so, why would we go through a proxy? (I think I know why, but it's worth clarifying.)

**Comment by kenoir** [comment]
Closing as outdated.

**Pull request closed without merging**